docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -343,6 +343,22 @@ The master supports the following configuration settings:
          compute resources, e.g. GPUs or dedicated CPUs. Defaults to the Slurm/PBS default partition
          if it has GPU resources and if no resource pool is specified.
 
+      -  ``job_project_source``: configures labelling of jobs on the HPC cluster (via Slurm
+         ``--wckey`` or PBS ``-P``). Allowed values are:
+
+         -  ``project``: Use the project name of the experiment (this is the default, if no project
+            nothing is passed to workload manager).
+
+         -  ``workspace``: Use the workspace name of the project (if no workspace, nothing is passed
+            to workload manager).
+
+         -  ``label[:prefix]``: Use the value from the experiment configuration labels list (if no
+            matching labels, nothing is passed to workload manager). If a label begins with prefix,
```

Comment:
Seem like prefix should be formatted to indicate it is the variable after the label.  At least ``prefix``.  Can we additionally make both prefixes italic as it is a variable provided by the user.